ArcadiaTeamsBot.ServiceDesk/ServiceDeskClient.cs

```diff
@@ -58,6 +59,24 @@ public Task<IEnumerable<ServiceDeskRequestPriorityDTO>> GetPriorities(Cancellati
             return Task.FromResult(this.priorities);
         }
 
+        public async Task CreateNewRequest(RequestForCreationDTO requestForCreationDTO, CancellationToken cancellationToken)
+        {
+            var httpRequest = new HttpRequestMessage(HttpMethod.Put, $"{this.serviceDeskConfiguration.ApiUrl}{createNewRequestUrl}");
+
+            httpRequest.Headers.Add("x-api-key", this.serviceDeskConfiguration.ApiKey);
+
+            var options = new JsonSerializerOptions
+            {
+                PropertyNameCaseInsensitive = true
```

Comment:
Doesn't needed. It does matter when you deserialize some string.

Reply:
Removed